devops/scripts/asa/scan.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
```

Comment:
Please add copyright notices to all new files

Reply:
Added, thanks!